lib/liftoff/project_configuration.rb

```diff
@@ -20,7 +20,8 @@ class ProjectConfiguration
       :xcode_command,
       :extra_config,
       :extra_test_config,
-      :deployment_target
+      :deployment_target,
+      :schemes
```

Comment:
But isn't the rule only for lists? Is this any different than https://github.com/thoughtbot/guides/blob/master/style/ruby/sample.rb#L19? (I'm not a Ruby developer, so forgive me if I'm writing something dumb)